test/e2e/util/data.go

```diff
@@ -1,44 +0,0 @@
-// Copyright 2020 Tetrate
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package util
-
-import (
-	"fmt"
-
-	"github.com/tetratelabs/getenvoy/pkg/extension/workspace/config/extension"
-)
-
-// CategoryLanguageTuple represents a combination of extension category and
-// programming language.
-type CategoryLanguageTuple struct {
-	extension.Category
-	extension.Language
-}
-
-func (t CategoryLanguageTuple) String() string {
-	return fmt.Sprintf("category=%s, language=%s", t.Category, t.Language)
-}
-
-// GetCategoryLanguageCombinations returns all combinations of a supported
-// extension category with a supported programming language.
-func GetCategoryLanguageCombinations() []CategoryLanguageTuple {
```

Comment:
I just thought that using env var here is enough for what we want to achieve here. Then we can have `E2E_EXTENSION_LANGUAGE=all` instead of `make e2e.all`. WDYT?

Reply:
something like
```go
if envVar == "all" || language == envVar { tuples = append(tuples, CategoryLanguageTuple{category, language}) }
```

Reply:
I think I will always override this to tinygo because it will take too long. Remember how it was annoying to have to set the go binary every time we run e2e in the IDE? This will put that back for the language..

What if we had make default that to all, but go default it to tinygo?

Reply:
right. Thanks for clarifying.